SampleApp/ViewController.m

```diff
@@ -23,6 +23,9 @@ - (void)viewDidLoad {
     UIButton *infoButton = [UIButton buttonWithType:UIButtonTypeInfoLight];
     [infoButton addTarget:self action:@selector(toggleOrientation:) forControlEvents:UIControlEventTouchUpInside];
     UIBarButtonItem *infoBarButton = [[UIBarButtonItem alloc] initWithCustomView:infoButton];
+    UIBarButtonItem *refreshButton = [[UIBarButtonItem alloc] initWithImage:[UIImage imageNamed:@"icon-refresh"] style:UIBarButtonItemStylePlain target:nil action:nil];
```

Comment:
Not sure if I'm missing something, but isn't this entirely unused?

Reply:
So it is, apologies! Fixing for you now ☺️

Reply:
…but this item still doesn't do anything – what's the motivation to add it here? Unless it demonstrates something crucial, I would rather leave out the added assets and keep this PR small.

 To me, only the changes to `ISHHoverBar.m` and `ISHHoverBarTests.m` are actually expected 😅

Reply:
Ahh gotcha!! This all happened a while ago sorry, I think my plan was to use it as an example for the new properties, but happy to remove it!